internal/pipe/git/git.go

```diff
@@ -290,6 +293,8 @@ func getPreviousTag(ctx *context.Context, current string) (string, error) {
 func gitTagsPointingAt(ctx *context.Context, ref string) ([]string, error) {
 	return git.CleanAllLines(git.Run(
 		ctx,
+		"-c",
+		"versionsort.suffix="+ctx.Config.Git.PrereleaseSuffix,
```

Comment:
I think we shouldn't assume `-` to be the default... maybe its better to only apply those when the config is not empty?